lib/pages/faq/view/faq_page.dart

```diff
@@ -0,0 +1,176 @@
+                      searchClosed = true;
+                      filter = '';
+                    });
+                  },
+                  searchClosed: searchClosed,
+                ),
+                QuestionsList(
+                    questions: questions
+                        .where((question) =>
+                            filter
+                                .split(' ')
+                                .where((element) => element != '')
+                                .fold(
+                                    true,
+                                    (previousValue, filter) =>
+                                        previousValue &&
+                                        question.question
+                                            .toLowerCase()
+                                            .contains(filter)) &&
+                            (activeTags.isEmpty
+                                ? true
+                                : containsTag(activeTags, question.tags)))
+                        .toList(),
+                    filter: filter),
+              ],
+            );
+          }),
+    );
+  }
+
+  bool containsTag(List<String> activeTags, List<String> questionTags) {
+    for (String tag in activeTags) {
+      if (questionTags.contains(tag)) {
+        return true;
+      }
+    }
+    return false;
+  }
+}
+
+class QuestionsList extends StatefulWidget {
+  final List<Question> questions;
+  final String filter;
+
+  QuestionsList({this.questions, this.filter});
+
+  @override
+  _QuestionsListState createState() => _QuestionsListState();
+}
+
+class _QuestionsListState extends State<QuestionsList> {
+  @override
+  Widget build(BuildContext context) {
+    List<String> filteredWords =
+        widget.filter.split(' ').where((element) => element != '').toList();
+    return Column(
+      children: [
+        Padding(
+          padding: EdgeInsets.only(top: 12.0),
+          child: ListView.builder(
+            shrinkWrap: true,
+            physics: NeverScrollableScrollPhysics(),
+            itemCount: widget.questions.length,
+            itemBuilder: (context, index) {
+              return ExpansionTile(
+                title: filteredWords.isNotEmpty
+                    ? DynamicTextHighlighting(
+                        text: widget.questions[index].question,
+                        style: Theme.of(context).textTheme.subtitle1,
+                        highlights: filteredWords,
+                        color: Theme.of(context).accentColor,
+                        caseSensitive: false,
+                      )
+                    : Text(
+                        widget.questions[index].question,
+                        style: Theme.of(context).textTheme.subtitle1,
+                      ),
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 15.0),
+                    child: MarkdownBody(
```

Comment:
Find a way to make it left-aligned when the content has less than one line (instead of center-aligned).